### PR TITLE
“Visual style – Colors”: Amend color palette overview & detailed swatches

### PR DESCRIPTION
**Pull request by Volker-E**
Amending color palette overview, reducing to useful amount of detailed
information. Updating detailed swatches and following discussion outcomes
on style and details provided.
Also moving paragraphs to be more consistent with overview.

Bug: [T167190](https://phabricator.wikimedia.org/T167190)
Bug: [T185567](https://phabricator.wikimedia.org/T185567)

### CODE REVIEW

File: css/build/wmui-style-guide.css [hunk 1]
```diff
@@ -977,31 +986,72 @@ sup {
...
+		margin-right: 1.35rem;
+	}
+.color-section .color .color-code--hsb {
+		-webkit-transition: opacity 250ms;
+		-moz-transition: opacity 250ms;
+		transition: opacity 250ms;
+	}
+.color-section .color:hover .color-code--hsb {
+			opacity: 1;
+		}
```
Comment (Ladsgroup) [inline css/build/wmui-style-guide.css]: extra tab

File: css/build/wmui-style-guide.css [hunk 2]
```diff
@@ -977,31 +986,72 @@ sup {
...
+		font-weight: 100;
+	}
+.color-palette--overview .color__name {
+			position: relative;
+			bottom: auto;
+			left: auto;
+			padding-top: 1.2rem;
+			font-size: 2.4rem;
+			line-height: 1;
+		}
```
Comment (Ladsgroup) [inline css/build/wmui-style-guide.css]: Same

File: css/build/wmui-style-guide.css [hunk 3]
```diff
@@ -977,31 +986,72 @@ sup {
...
+.color__type,
+		.color-code.color-code--rgb,
+		.color-palette--overview .color__name,
+		.color-palette--overview .color-code {
+			display: none;
+		}
```
Comment (Ladsgroup) [inline css/build/wmui-style-guide.css]: Same

File: css/build/wmui-style-guide.css [hunk 4]
```diff
@@ -977,31 +986,72 @@ sup {
...
 .color-code {
 		background-color: inherit;
 		display: block;
 		padding: 0;
 		font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Oxygen-Sans', Ubuntu, Cantarell, Lato, 'Helvetica Neue', Helvetica, Arial, sans-serif;
 	}
+.color-code--hsb {
+			opacity: 0;
```
Comment (Ladsgroup) [inline css/build/wmui-style-guide.css]: same :D

**Comment by Volker-E** [comment]
@Ladsgroup Thanks, I'll make your comment a topic again with the other design team members.

**Pull request merged**